R/restrict_derivation.R

```diff
@@ -6,6 +6,11 @@
 #'
 #' @param derivation Derivation
```

Comment:
What is a Derivation?  Could we say something about how the function is performing the derivation.  I know what this is doing I just think a new user might be confused

Reply:
Good point! I have added a short definition.

In principle a glossary would be nice but hard to realize in R documentation.

Reply:
Omg a glossary would be fantastic.  I'm making an issue for this lol.